src/App.js

```diff
@@ -10,35 +10,37 @@ function App() {
   const [selectedMovieID, setSelectedMovieID] = useState("");
   const [apiMovieData, setApiMovieData] = useState([]);
   const [selectedMovieObj, setSelectedMovieObj] = useState({});
-  const [error, setError] = useState(null)
+  const [selectedMovieTrailerLink, setSelectedMovieTrailerLink] = useState("");
+  const [error, setError] = useState(null);
 
- 
   function getMoviesFromApi() {
     fetch("https://rancid-tomatillos.herokuapp.com/api/v2/movies")
       .then((res) => {
         if (!res.ok) {
-          throw new Error("Oops! Something went wrong on the server. Please try again later.");
+          throw new Error(
```

Comment:
Are you planning to throw errors, or use state to show errors to your user? I would pick on and stay consistent with it!